sqladmin/models.py

```diff
@@ -712,6 +724,14 @@ def _url_for_details_with_attr(self, obj: Any, attr: RelationshipProperty) -> st
             pk=pk,
         )
 
+    def _url_for_action(self, obj: Any, action_name: str) -> str:
+        pk = getattr(obj, inspect(obj).mapper.primary_key[0].name)
```

Comment:
I think the object passed here is an instance of the current ModelView, so maybe this should work:
```python
pk = getattr(obj, self.pk)
```

Reply:
or maybe `getattr(obj, get_primary_key(obj).name)` like in other `_url_for_delete/_url_for_edit/_url_for_details` ?

Reply:
My point was to avoid calling `inspect` again, but `get_primary_key()` will do that.
I think my comment should've been `pk = getattr(obj, self.pk.name)`, doesn't this work?

Reply:
hoo ok, i don't see any `pk` attribute ... or maybe i'm not looking the good object

Reply:
my bad it's `getattr(obj, self.pk_column.name)` not `pk`